modules/articles/client/views/admin/form-article.client.view.html

```diff
@@ -3,7 +3,7 @@
     <h1>{{vm.article._id ? 'Edit Article' : 'New Article'}}</h1>
   </div>
   <div class="pull-right">
-    <a class="btn btn-primary" ng-click="vm.remove()">
+    <a class="btn btn-primary" ng-click="vm.remove()" ng-if="!vm.isNew">
```

Comment:
This is unrelated to what this PR is trying to accomplish, but I noticed a bug while developing this new feature. The "Delete" button was visible when the user was in "create" mode here.